cmd/frontrunning/main.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"math/big"
+	"net/http"
+	"time"
+
+	"github.com/Amenokal-Labs/mev-mantis.git/pkg/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/ethclient/gethclient"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func main() {
+	ethclient, err := ethclient.Dial("https://goerli.infura.io/v3/" + utils.GetKey("INFURA_KEY"))
+	if err != nil {
+		log.Fatal("[0] ", err)
+
+	}
+
+	rpc, err := rpc.Dial("wss://goerli.infura.io/ws/v3/" + utils.GetKey("INFURA_KEY"))
+	if err != nil {
+		log.Fatal("[1] ", err)
+	}
+	subscriber := gethclient.New(rpc)
+	fmt.Println("client connected..")
+
+	// txs := make(chan *types.Transaction)
+	// _, err = rpcClient.SubscribeFullPendingTransactions(context.Background(), txs)
+	// if err != nil {
+	// 	panic(err)
+	// }
+
+	hashes := make(chan common.Hash)
+	_, err = subscriber.SubscribePendingTransactions(context.Background(), hashes)
+	if err != nil {
+		log.Fatal("[2] ", err)
+	}
+
+	for {
+		pc, _ := ethclient.PendingTransactionCount(context.Background())
+		fmt.Println("\nPending count:", pc)
+
+		hash := <-hashes
+		fmt.Println("      Tx hash:", hash)
```

Comment:
you can replace these whitespaces with `\t`, like: `fmt.Println("\tTx hash:", hash)`

Reply:
This didn't do it because i need a different amount of spaces for each line to get this output
![image](https://github.com/Amenokal-Labs/mev-mantis/assets/128217762/c111d590-e8f5-497a-8ed1-e3e7bdf5dc32)

Reply:
oh ok, it makes sense